service/controller/clusterapi/v29/resource/tccp/testdata/case-0-basic-test.golden

```diff
@@ -4,7 +4,7 @@ Outputs:
   DockerVolumeResourceName:
     Value: rsc-abbacd01
   MasterImageID:
-    Value: ami-0eb0d9bb7ad1bd1e9
+    Value: 
```

Comment:
And look, I fixed something. 🙈

Reply:
Ah, the heat. After my run I came back to realize this did not fix but break something. Fixing.